get-resource.sh

```diff
@@ -10,10 +10,12 @@ export no_proxy="${no_proxy:-${NO_PROXY:-}}"
 
 # configurable variables
 SHARED_DIR="${SHARED_DIR:-/shared}"
-CURL_OUTPUT=""
-if [ "${CURL_VERBOSE:-}" = true ]; then
-    CURL_OUTPUT="--verbose"
-fi
+
+curl_with_flags() {
+    [ "${CURL_VERBOSE:-}" != 'true' ] || set -- --verbose "$@"
+    [ "${CURL_INSECURE:-}" != 'true' ] || set -- --insecure "$@"
+    curl "$@"
```

Comment:
What was wrong with

```
if [ "${CURL_INSECURE:-}" = true ]; then
    CURL_OUTPUT="${CURL_OUTPUT} --insecure"
fi
```

?

Reply:
@zaneb shellcheck was [complaining](https://prow.apps.test.metal3.io/view/s3/prow-logs/pr-logs/pull/metal3-io_ironic-ipa-downloader/54/shellcheck/1780915098823430144) for https://github.com/metal3-io/ironic-ipa-downloader/commit/e0bd9e3ca22e77d969167f6bb36b121b457c7e6a.

See https://www.shellcheck.net/wiki/SC2086